.github/workflows/tests.yml

```diff
@@ -18,7 +18,6 @@ jobs:
         with:
           profile: minimal
           toolchain: stable
-          target: x86_64-unknown-linux-musl
```

Comment:
I also removed this line since it's clearly a bad copypasta. There is a musl-specific test below in the file which needs a musl target. This one here runs on a matrix of operating systems, not all of which are Linux.